docs/data_format_changes/i3477-version-query-consistency.md

```diff
@@ -0,0 +1,2 @@
+# Consistent version query semantics
+Updated semantics for commit/version/time-travel queries when using both DocID and CID.
```

Comment:
todo: There is no data breaking change in this PR, if the change detector failed it is because the integration test definitions were changed, not the data on disk.  Please change the wording to reflect this.